### ISSUE TITLE
jquery dependency

### ISSUE THREAD

**Issue by miguelalarcos** [Anonyfox/mmenu#1]
hi, I have had to add jquery to my project to work with this package. I think the package should include jquery as a dependency.

**Comment by Anonyfox** [comment]
AFAIK meteor does require query (and underscore) by default, no?

**Comment by miguelalarcos** [comment]
Yes, you are right when you are writing an app. But when you are writing a package, as far as a I know, you have to specify it.
